torch_xla/csrc/tensor.cpp

```diff
@@ -1283,6 +1284,8 @@ std::shared_ptr<XLATensor::Async> XLATensor::ScheduleSyncTensorsGraph(
     std::vector<xla::ComputationClient::DataPtr> parameters_data,
     std::vector<xla::ComputationClient::DataPtr> tensors_data,
     ComputationCache::TypePtr cached_computation) {
+  tensorflow::profiler::TraceMe activity(
```

Comment:
Maybe also add trace to `CollectSyncTensors ` and `SyncTensorsGraphInternal `. They are more of helper functions but are used by many methods and might be slow.

Reply:
Added them to those as well. FYI, adding these traces should be pretty simple going forward so let's sprinkle them around our stack going forward where we see fit 😄 and tune the verbosity if needed.